terraform/aws/modules/drbd_node/variables.tf

```diff
@@ -18,40 +18,39 @@ variable "drbd_count" {
   default     = 2
 }
 
-variable "instance_type" {
+variable "vm_size" {
   description = "The instance type of drbd node"
   type        = string
-  default     = "t2.large"
```

Comment:
Is this default used anywhere, or can it be safely removed?

Reply:
All the 3 cloud providers has the same layered architecture:
- main layer: it has a main and a variable file. The main in this layer take variables in this layer and use them to configure variables for each module in the next layer.
- modules: each module has its main and its variable file

Now use this model about variables to configure the VM type:
- **AZURE**: 
  1. Example of top level variable https://github.com/SUSE/qe-sap-deployment/blob/f3e2b04a9af57e5eba2004fde18f13949b2e33b3/terraform/azure/variables.tf#L202 . It has a default.
  2. Example of top level main that configure module layer low level https://github.com/SUSE/qe-sap-deployment/blob/f3e2b04a9af57e5eba2004fde18f13949b2e33b3/terraform/azure/main.tf#L204
  3. Low level variable: https://github.com/SUSE/qe-sap-deployment/blob/f3e2b04a9af57e5eba2004fde18f13949b2e33b3/terraform/azure/modules/hana_node/variables.tf#L62  It still has a default, but we can think about removing it (like I started doing in AWS https://github.com/SUSE/qe-sap-deployment/pull/261/files#diff-5e0f222194e1a8dbb339d9203a8a956ba75c2d44405c68ebd4435807f01a369eL23)
  4. Low level main where the variable value is used when finally create the VM https://github.com/SUSE/qe-sap-deployment/blob/f3e2b04a9af57e5eba2004fde18f13949b2e33b3/terraform/azure/modules/hana_node/main.tf#L392 .